releasenotes/notes/0.5-prelude-a1d20e9b76893ece.yaml

```diff
@@ -0,0 +1,111 @@
+---
+prelude: >
+    Qiskit Nature 0.5 comes with a major redesign of the :class:`.BaseProblem`
+    layer of its stack. Rather than being tightly integrating both, drivers and
+    transformers, problems are now a lot more standalone containers, being
+    generated by the various drivers (or built out by a user to their custom
+    needs directly). As a short example, compared to the old initialization:
```

Comment:
```suggestion
    layer of its stack. Rather than tightly integrating drivers and transformers,
    problems are now a lot more standalone and are generated by the
    various drivers (or built out by a user to their custom needs directly).
    As a short example comparing the previous to the new problem creation:
```